### ISSUE TITLE
Issue with RealConfig.bat

### ISSUE THREAD

**Issue by 0Pink88** [LukeRoss00/gta5-real-mod#88]
Hello! Whenever I run RealConfig.bat I get error messages saying 
The system cannot find the path specified.
The system cannot find the path specified.
        0 file(s) copied.
        1 file(s) copied.
I tried yesterday to fix this and could not get it to work.
I have GTA installed on my D: drive and operating system is on my C: drive if any of that matters
HTC VIVE
1080 ti
(I am also trying to select medium settings)

**Comment by LukeRoss00** [comment]
Did you check the README?

> **If the batch file gives you errors like "The system cannot find the path specified" or "0 file(s) copied":** The `RealConfig.bat` file assumes that your Documents folder is at `%UserProfile%\Documents`. If you moved or redirected it, you should manually edit the batch file in Notepad and replace `%UserProfile%\Documents` with the full path of your Documents folder, then run the batch file again.

**Comment by jeq1** [comment]
I have this same problem, i havent moved documents folder and it is there. My documents folder doesn't include any gta V files tho. So where i now find right directory?

**Comment by chalmers78** [comment]
I'm having the same issue now, where did you find it?

**Comment by 0Pink88** [comment]
> I'm having the same issue now, where did you find it?

I tried copying the address of my documents folder and pasting it into the dst= in the bat file. It seems the issue has changed. It now says
"the system cannot find the file specified
               1 file(s) copied.
               1 file(s) copied.
Not sure whats wrong now

**Comment by LukeRoss00** [comment]
@0Pink88 Please list the contents of your `Documents\Rockstar Games\GTA V` folder. You can also perform the operations manually as explained [here](https://github.com/LukeRoss00/gta5-real-mod/issues/76#issuecomment-582357234)

**Comment by 0Pink88** [comment]
Okay so here is the contents of my Documents folder: https://gyazo.com/dcff2ab0530ff31de9b4da8a73d6e6be

**Comment by LukeRoss00** [comment]
Looks OK. But did you run the game again after you fixed the batch file with the path to your Documents folder? What problems does the game have?

**Comment by 0Pink88** [comment]
Seems my game is just constantly crashing while loading up now.

**Comment by LukeRoss00** [comment]
Please post your `asiloader.log` and `ScriptHookV.log` files

**Comment by 0Pink88** [comment]
[asiloader.log](https://github.com/LukeRoss00/gta5-real-mod/files/4176491/asiloader.log)
[ScriptHookV.log](https://github.com/LukeRoss00/gta5-real-mod/files/4176493/ScriptHookV.log)

**Comment by LukeRoss00** [comment]
The game appears to crash very early, before my script is even loaded. One last thing we can try: add the line `RVRLog = 2` at the end of your `RealVR.ini` and try launching the game again. After the crash you might have a new `RVRLog.txt` file, please attach it here

**Comment by 0Pink88** [comment]
[RVRLog.txt](https://github.com/LukeRoss00/gta5-real-mod/files/4177674/RVRLog.txt)

**Comment by LukeRoss00** [comment]
Well, as you can see the game crashes immediately (there is a single line in the log). Not sure what's happening as you are the only user that has this early crash issue. Does the game run correctly if you uninstall the mod, that is remove all the files that the mod put in?

**Comment by 0Pink88** [comment]
Yes, I just uninstalled the mod and the game runs perfectly fine. Not sure what's happening.

**Comment by LukeRoss00** [comment]
It's possible that some other software is interfering. Do you have stuff like overlays, other mods, utilities, screen grabbers, overzealous antivirus programs, etc. that might be preventing the mod from loading? Any non-default settings in the NVIDIA drivers, either globally or for the game?

**Comment by JesusBourne** [comment]
my logs are exaclty the same as his. and i have the same problem

**Comment by JesusBourne** [comment]
I just found the solution for this problem. it has to do with the config.bat file. if you run this and get the message file not found but both files where copied you are screwed. you have to restart. basically undo what the bat file did. what i did was i changed my documents location form onedrive to where the bat file wanted it users\micha\documents(this wasnt as easy as expected). next i uninstalled the mod in the game files and the doc files. reinstalled  the mod, opened the bat file and it worked first try. i then started the game and it worked. 

long story short i restarted after an error (the file was not found and both files where copied) and it worked.

also ik this is confusing. just ask questions:)
